src/compiler/utilities.ts

```diff
@@ -1884,8 +1884,8 @@ namespace ts {
             writeTextOfNode,
             writeLiteral,
             writeLine,
-            increaseIndent: () => indent++,
-            decreaseIndent: () => indent--,
+            increaseIndent: () => { indent++; },

```

Comment:
It doesn't matter -- `EmitTextWriter` says these functions are void, so callers cannot have used the return value.

(Yes, we are saner than we look!)